[CI]: lima (experimental/fedora-rawhide) (`which: command not found`)

There is an error in "Run integration tests (fs driver, rootless)":
```
Run ssh -tt lima-default sudo -i make -C /tmp/runc localrootlessintegration
]3008;start=ce04f5340efe44cda03fe2be6b7b7ede;user=runner;hostname=lima-default;machineid=176e17a253c04a3b9ca7a88c94927859;bootid=f57e361dd19c4afd8130728ef03c42dd;pid=169483;pidfdid=173435;comm=sudo;targetuser=root;type=session\make: Entering directory '/tmp/runc'
go build -trimpath "-buildmode=pie"  -tags "seccomp urfave_cli_no_docs libpathrs " -ldflags "-X main.gitCommit=8b55192  " -o runc .
go build -trimpath "-buildmode=pie"  -tags "seccomp urfave_cli_no_docs libpathrs " -ldflags "-X main.gitCommit=8b55192  " -o tests/cmd/_bin ./tests/cmd/recvtty
go build -trimpath "-buildmode=pie"  -tags "seccomp urfave_cli_no_docs libpathrs " -ldflags "-X main.gitCommit=8b55192  " -o tests/cmd/_bin ./tests/cmd/sd-helper
go build -trimpath "-buildmode=pie"  -tags "seccomp urfave_cli_no_docs libpathrs " -ldflags "-X main.gitCommit=8b55192  " -o tests/cmd/_bin ./tests/cmd/seccompagent
go build -trimpath "-buildmode=pie"  -tags "seccomp urfave_cli_no_docs libpathrs " -ldflags "-X main.gitCommit=8b55192  " -o tests/cmd/_bin ./tests/cmd/fs-idmap
go build -trimpath "-buildmode=pie"  -tags "seccomp urfave_cli_no_docs libpathrs " -ldflags "-X main.gitCommit=8b55192  " -o tests/cmd/_bin ./tests/cmd/pidfd-kill
go build -trimpath "-buildmode=pie"  -tags "seccomp urfave_cli_no_docs libpathrs " -ldflags "-X main.gitCommit=8b55192  " -o tests/cmd/_bin ./tests/cmd/remap-rootfs
go build -trimpath "-buildmode=pie"  -tags "seccomp urfave_cli_no_docs libpathrs " -ldflags "-X main.gitCommit=8b55192  " -o tests/cmd/_bin ./tests/cmd/key_label
tests/rootless.sh
[01] run rootless tests ... ()
path: /root/.local/bin:/root/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/var/lib/snapd/snap/bin
tests/rootless.sh: line 218: which: command not found
sudo: : command not found
make: *** [Makefile:189: localrootlessintegration] Error 1
make: Leaving directory '/tmp/runc'
Shared connection to 127.0.0.1 closed.
]3008;end=ce04f5340efe44cda03fe2be6b7b7ede\
Error: Process completed with exit code 2.
```

--- a/tests/rootless.sh
+++ b/tests/rootless.sh
@@ -215,7 +215,7 @@ for ROOTLESS_FEATURES in $features_powerset; do
 			rootless@localhost -- "${ssh_env[@]}" bats -t "$ROOT/tests/integration$ROOTLESS_TESTPATH"
 	else
 		export "${ENV_LIST[@]}"
-		sudo -HE -u rootless PATH="$PATH" "$(which bats)" -t "$ROOT/tests/integration$ROOTLESS_TESTPATH"
+		sudo -HE -u rootless PATH="$PATH" "$(command -v bats)" -t "$ROOT/tests/integration$ROOTLESS_TESTPATH"
 	fi
 	cleanup
 done

